src/components/Footer.jsx

```diff
@@ -24,8 +24,8 @@ export function Footer() {
           >
             {[
               [
```

Comment:
These changes look irrelevant, please make them in a different PR

Reply:
@xkrishguptaa what exactly I need to do. As I am understanding I need to make another PR for the footer changes am right?

Reply:
@ninadvyas you are fixing two issues right? Please keep this PR only for the EddieHub Mascot fix and do the links of partner's fix in a different PR

Reply:
I am fixing two issues. Can you teach me how to make different PR because I have committed changes in my main branch and according to me I have to make another branch for another PR. Is that correct? @xkrishguptaa